contrib/build-wine/deterministic.spec

```diff
@@ -10,7 +10,7 @@ for i, x in enumerate(sys.argv):
 else:
     raise BaseException('no name')
 
-
+PYHOME = 'c:/python3.6.8'
```

Comment:
Hmm. Any chance you can figure out how to do this without hard-coding `3.6.8`?  We just recently got away from the nightmare of having hard-coded versions of everything (including python). It used to be you'd have to edit 4,000 files every time we switch python version and you'd always miss one.  We don't want to slowly decay back to that level of entropy..  Any chance you can glob the dir name from `c:\\python*` or something?

Reply:
I found an easy way to remove the hard-coded version number. I will provide the change in a commit this week.